hardhat.config.js

```diff
@@ -21,6 +21,10 @@ module.exports = {
       accounts: {
         count: 1,
       },
+      mining: {
+        auto: false,
+        interval: 8000,
```

Comment:
why not 12_000?
also, pls format as X_000 for readability.

Reply:
12_000 seems like a good default, but I was concerned about making these tests too long, so went with a slightly shorter default.

Reply:
this won't have auto-mining, so this should just configure the apparent time between blocks. ie if you inspect the timestamp on blocks, they'll be `interval` seconds apart, but they'll _actually_ just mine when you tell them to, they shouldn't add that delay to these tests.

I think you should use `12000` because that's the expected block time on mainnet.

Reply:
I'm 100% wrong here - I just checked the docs: https://hardhat.org/hardhat-network/docs/reference#mining-modes.

Why do we want interval mining at all? Why not just mine when we call `hardhat.mine()`?